src/librustc_resolve/lib.rs

```diff
@@ -953,6 +981,19 @@ impl<'a> ModuleS<'a> {
         assert!(self.pub_glob_count.get() > 0);
         self.pub_glob_count.set(self.pub_glob_count.get() - 1);
     }
+
+    fn is_successor_of(&self, ancestor: Module<'a>) -> bool {
```

Comment:
In fact, following the suggestion of Niko, it would be better to show all candidates from the current crate, since the programmer can easily make something public. WDYT?